national-resources/switzerland.md

```diff
@@ -0,0 +1,79 @@
+* National Notification System: The FOPH National Notification System requires healthcare providers to report cases of certain infectious diseases to the FOPH. This system enables the FOPH to track the spread of diseases, identify outbreaks, and initiate public health responses.
+* Laboratory surveillance: Switzerland has an extensive network of National Reference Centers that are involved in disease surveillance, including the CRIVE - the National Reference Center for Influenza and Emerging Viruses, and ANRESIS - the Swiss Centre for Antibiotic Resistance surveillance. These laboratories conduct diagnostic tests and monitor trends in disease activity.
+Hospital based surveillance of Influenza and COVID-19 case (CH-SUR).
+* Travel health surveillance: Switzerland also has a system for monitoring the health of travellers, particularly those returning from high-risk areas. The system involves screening travellers for symptoms of infectious diseases and providing them with information and guidance on how to prevent the spread of disease.
+
+All these surveillance initiatives are funded by the FOPH and coordinated by them or in collaboration with institutions and laboratories. In the specific case of COVID-19, all the national monitoring efforts are summarised [here](https://www.bag.admin.ch/bag/en/home/krankheiten/ausbrueche-epidemien-pandemien/aktuelle-ausbrueche-epidemien/novel-cov/situation-schweiz-und-international/monitoring.html).
+
+## Dashboards and visualisation platforms
+
+* [Dashboard COVID-19 Switzerland FOPH  overview](https://www.covid19.admin.ch/en/overview)
+* [COVID-19 Data Portal direct access to Swiss COVID-19-related data](https://www.covid19dataportal.org/search?query=switzerland&requestFrom=searchBox )
+
+
+## National data sources
+<!--- A section to list and provide context to national data sources. In the context of BY-COVID, a data source can be a repository which should include at least the metadata and ideally the data, that might not be directly available when considering sensitive data. Also, repositories should have the capacity to share this data and therefore have a governance model in place on how to do it. It can also include registries of data sources important for the field, with a direct link to the original data sources to be able to request access to the data. --->
+
+
+* [SPSP](https://www.spsp.ch/) - A secure One-health online platform that enables near real-time sharing under controlled access of pathogen whole genome sequences (WGS) and their associated clinical/epidemiological metadata. Since 2021 it has centralized and processed all SARS-CoV-2 sequencing data within the national genomic surveillance program. 
+* [Swiss Biobank](https://swissbiobanking.ch/) - one line description
```

Comment:
Misses the description

Reply:
thanks! I have added that sentence, and some links that were missing elsewhere in that paragraph.